tests/utils/verification/volumeproperties.go

```diff
@@ -97,6 +100,75 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// GetContainerName returns full container name based on given short name
+func GetContainerName(hostName, shortName string) (string, error) {
+	cmd := dockercli.ListContainers + "--filter name='" + shortName + "' --format '{{.Names}}'"
```

Comment:
I think we should create filter constants for docker volume properties .. e.g. `--format '{{.Names}}` .. it will be easier to make change in one shot to sync with docker framework's change.